tests/test_healthsystem.py

```diff
@@ -193,6 +193,58 @@ def test_run_no_interventions_allowed(tmpdir, seed):
     assert not any(sim.population.props['mi_status'] == 'P')  # No cures
 
 
+@pytest.mark.slow
+def test_policy_has_no_effect_on_mode1(tmpdir, seed):
+    # Events ran in mode 1 should be identical regardless of policy assumed.
+    # In policy "No Services", have set all HSIs to priority below lowest_priority_considered,
+    # in mode 1 they should all be scheduled and delivered regardless
+
+    output = []
+    for i, policy in enumerate(["Naive", "No Services"]):
```

Comment:
why not iterate through all the policies?

Reply:
I think iterating through all policies is unnecessary, and the test is already quite slow, especially if considering a full model. The comparison between two priority policies can already test all the differences we are after (i.e. different priority of services both above and below the lowest_priority_considered).